src/pages/master-teachers-table/master-teachers-table.jsx

```diff
@@ -22,47 +52,114 @@ const MasterTeacherTableView = () => {
+        teacherObj;
+      const id = userId || inviteId;
+      const name = `${lastName}, ${firstName}`;
+      allTeachers.push({
+        id,
+        items: [
+          name,
+          { email, phoneNumber },
+          sites || [],
+          teacherObj, // used to show active status and resend invite if needed
+          null, // empty placeholder for reset password to make sure there is something for each column
+          notes || '',
+        ],
+        sortBy: name,
+      });
+    });
+    setBodyData(allTeachers);
+  };
+
+  const fetchData = async () => {
+    const teacherData = [];
+
+    // fetching all teacher accounts (created and invited)
+    const admins = await TLPBackend.get(`/teachers`, {
       headers: {
         'Content-Type': 'application/json',
       },
     });
-    if (res.status === 200) {
-      setMasterTeacherList(res.data);
-    } else {
-      setMasterTeacherList([]);
-      setError(error);
+    if (admins.status === 200) {
+      teacherData.push(...admins.data);
     }
+    parseTableData(teacherData);
+  };
+
+  useEffect(() => {
+    fetchData();
   }, []);
 
-  const tbodyData = [];
-  masterTeacherList.forEach(mtObj => {
-    const { firstName, lastName } = mtObj;
-    tbodyData.push({
-      id: mtObj.userId,
-      items: [`${firstName} ${lastName}`, mtObj.email, mtObj.sites, mtObj.active, 'notes'],
-    });
-  });
   return (
     <div className={styles['master-teacher-container']}>
-      <ManagementDataSection
-        sectionTitle="Master Teachers"
-        theadData={theadData}
-        tbodyData={tbodyData}
-        tbodyColIsBadge={[2]}
-        statusCol={3}
-      />
+      <div className={styles['table-view']}>
+        <div className={styles['table-header']}>
+          <h3>Master Teacher</h3>
+          <Button className={styles['export-button']} variant="primary">
+            Export to CSV
+          </Button>
+        </div>
+        <div className={styles['table-buttons']}>
+          <Button className={styles['create-button']} variant="warning" onClick={createTeacher}>
+            Create New Teacher Account <FaPlus cursor="pointer" />
+          </Button>
+          <InputGroup>
+            <FormControl
+              placeholder="Search Teachers"
+              aria-label="Search Teachers"
+              aria-describedby="search-icon"
+            />
+          </InputGroup>
+          <Button className={styles['filter-button']} variant="primary">
+            Filter By <FaFilter cursor="pointer" />
+          </Button>
+          <DropdownButton
+            className={styles['dropdown-button']}
+            id="dropdown-basic-button"
+            title={`Sort By ${sortBy}`}
+          >
+            <Dropdown.Item onClick={() => updateSortBy('A-Z')}>A-Z</Dropdown.Item>
+            <Dropdown.Item onClick={() => updateSortBy('Z-A')}>Z-A</Dropdown.Item>
+          </DropdownButton>
+        </div>
+        <Table
```

Comment:
It doesn't seem like there's a way to remove sites right now through the badge?

![image](https://user-images.githubusercontent.com/57849152/168151261-4819cf9f-9f87-43ab-b5bc-c3af99a7c680.png)

Reply:
This changed because Meredith wanted a way to show the admin the summary of their editing sites (via this [modal](https://www.figma.com/file/O2igPzMS1fZPzjmOxlo6Mc/TLP-Wireframes?node-id=4526%3A121076)) and with the option of deleting sites at the cell (it would mean alerts for every delete or some way to turn of alert notifs), so we decided to just change it to open the edit modal to signify editing from the modal instead.

Reply:
ah got it